activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -20,18 +20,12 @@ def initialize
       @odd = false
     end
 
-    def render_bind(attribute)
-      value = if attribute.type.binary? && attribute.value
-        if attribute.value.is_a?(Hash)
-          "<#{attribute.value_for_database.to_s.bytesize} bytes of binary data>"
-        else
-          "<#{attribute.value.bytesize} bytes of binary data>"
-        end
-      else
-        type_cast(attribute.value_for_database)
+    def render_bind(attr, value)
```

Comment:
Should the second argument be called "typed_casted_value" for coherence? As a reader I wonder why there is a `value` passed but ignored because `attr.value` is used internally.